docusaurus/src/components/Details.module.css

```diff
@@ -0,0 +1,41 @@
+.details {
+  --docusaurus-details-decoration-color: var(--ifm-color-info-dark);
+  --docusaurus-details-transition: transform var(--ifm-transition-fast) ease;
```

Comment:
One different I noticed between this and the original docusaurus `details` is that the docusaurus one would animate a "slide down" of the content when opening, and "slide up" when closing, but this custom one just immediately opens or closes the full content.

I don't think this is a huge deal if it is hard to get working right, but just wanted to point it out

Reply:
I tried to copy the behaviour but can't make it work. I'll add this in a new issue as a nice-to-have to unblock this one :)